lib/js/components/RichList/RichListGroupItem/RichListGroupItem.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+	<div
+		class="richListGroupItem"
+		:class="{
+			[borderColorClass]: !!borderColor,
+		}"
+		:style="{
+			borderColor: borderColorHex ? borderColorHex : null,
+		}"
+	>
+		<div class="richListGroupItem__header">
+			<slot name="header" />
+		</div>
+		<div v-if="isExpanded" class="richListGroupItem__expanded">
+			<slot name="expanded" />
+		</div>
+		<div class="richListGroupItem__border" />
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../../styles/settings/colors/tokens';
+@import '../../../../styles/settings/radiuses';
+@import '../../../../styles/settings/spacings';
+
+$rich-list-group-item-border-colors: (
+	default: $color-default-border,
+	primary: $color-primary-border,
+	primary-weak: $color-primary-border-weak,
+	neutral-heavy: $color-neutral-border-heavy,
+	neutral-strong: $color-neutral-border-strong,
+	neutral: $color-neutral-border,
+	neutral-weak: $color-neutral-border-weak,
+	neutral-ghost: $color-neutral-border-ghost,
+	danger: $color-danger-border,
+	danger-weak: $color-danger-border-weak,
+	fail: $color-fail-border,
+	fail-weak: $color-fail-border-weak,
+	warning: $color-warning-border,
+	warning-weak: $color-warning-border-weak,
+	success: $color-success-border,
+	success-weak: $color-success-border-weak,
+	info: $color-info-border,
+	info-weak: $color-info-border-weak,
+	accent: $color-accent-border,
+	accent-weak: $color-accent-border-weak,
+);
+
+.richListGroupItem {
+	$root: &;
+
+	@each $color, $value in $rich-list-group-item-border-colors {
+		&.-border-#{$color} {
+			#{$root}__border {
+				background-color: $value;
+			}
+		}
+	}
+
+	border: 1px solid $color-neutral-border-weak;
+	border-radius: $radius-s;
+	display: flex;
+	flex: 1;
+	flex-direction: column;
+	justify-content: center;
+	max-width: 100%;
+	overflow: hidden;
+	position: relative;
+
+	&__header {
+		background: $color-neutral-background;
+	}
+
+	&__border {
+		border-radius: $radius-s 0 0 $radius-s;
+		height: calc(100% + 2px);
+		left: -1px;
+		position: absolute;
+		top: -1px;
+		width: $space-xxxxs;
+	}
+}
+</style>
+
+<script lang="ts">
+import { PropType } from 'vue';
+import {
+	RICH_LIST_GROUP_ITEM_BORDER_COLOR,
+	RichListGroupItemBorderColor,
+} from './RichListGroupItem.consts';
+
+export default {
+	name: 'RichListGroupItem',
+	components: {},
+	props: {
```

Comment:
So I think about if all the other props are needed, and probably they are not. But should we construct a story the way we can, with one click, change ex `isDimmed` prop of all children and the parent?

Reply:
I thought about it, but if I add a prop in this component that overwrites others, it may be confusing that the group also has such a prop. That's why I rejected it.